src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -47,96 +50,68 @@ public FeedbackSessionAttributes() {
-        this.sentPublishedEmail = sentPublishedEmail;
-        this.isOpeningEmailEnabled = isOpeningEmailEnabled;
-        this.isClosingEmailEnabled = isClosingEmailEnabled;
-        this.isPublishedEmailEnabled = isPublishedEmailEnabled;
-        this.respondingInstructorList = instructorList;
-        this.respondingStudentList = studentList;
+        this.respondingInstructorList = new HashSet<>();
+        this.respondingStudentList = new HashSet<>();
+    }
+
+    public static FeedbackSessionAttributes valueOf(FeedbackSession fs) {
+        return builder(fs.getFeedbackSessionName(), fs.getCourseId(), fs.getCreatorEmail())
+                .withInstructions(fs.getInstructions())
+                .withCreatedTime(fs.getCreatedTime())
+                .withStartTime(fs.getStartTime())
+                .withEndTime(fs.getEndTime())
+                .withSessionVisibleFromTime(fs.getSessionVisibleFromTime())
+                .withResultsVisibleFromTime(fs.getResultsVisibleFromTime())
+                .withTimeZone(fs.getTimeZone())
+                .withGracePeriod(fs.getGracePeriod())
+                .withFeedbackSessionType(fs.getFeedbackSessionType())
+                .withSentOpenEmail(fs.isSentOpenEmail())
+                .withSentClosingEmail(fs.isSentClosingEmail())
+                .withSentClosedEmail(fs.isSentClosedEmail())
+                .withSentPublishedEmail(fs.isSentPublishedEmail())
+                .withOpeningEmailEnabled(fs.isOpeningEmailEnabled())
+                .withPublishedEmailEnabled(fs.isPublishedEmailEnabled())
+                .withRespondingInstructorList(fs.getRespondingInstructorList())
+                .withRespondingStudentList(fs.getRespondingStudentList())
+                .withClosingEmailEnabled(fs.isClosingEmailEnabled())
+                .build();
     }
 
-    private FeedbackSessionAttributes(FeedbackSessionAttributes other) {
-        this(other.feedbackSessionName, other.courseId, other.creatorEmail,
-            other.instructions, other.createdTime, other.startTime, other.endTime,
-            other.sessionVisibleFromTime, other.resultsVisibleFromTime, other.timeZone,
-            other.gracePeriod, other.feedbackSessionType,
-            other.sentOpenEmail, other.sentClosingEmail, other.sentClosedEmail, other.sentPublishedEmail,
-            other.isOpeningEmailEnabled, other.isClosingEmailEnabled,
-            other.isPublishedEmailEnabled, other.respondingInstructorList,
-            other.respondingStudentList);
+    /**
+     * Return new builder instance with default values for optional fields.
+     *
+     * <p>Following default values are set to corresponding attributes:
+     * {@code isOpeningEmailEnabled = true} <br>
+     * {@code isClosingEmailEnabled = true} <br>
+     * {@code isPublishedEmailEnabled = true} <br>
+     * {@code respondingInstructorList = new HashSet<>()} <br>
+     * {@code respondingStudentList = new HashSet<>()} <br>
```

Comment:
Instead of `<p>` and `<br>` to indicate line breaks, consider using lists (`<ul>` and `<li>`).